### ISSUE TITLE
detect embeded streaming videos

### ISSUE THREAD

**Issue by trimechee** [setvisible/DownZemAll#102]
Hello, thank you so much for this wonderful download manger ! 💯 
I have enhancement suggestion please : we konow that yt-dlp support many video platforms but some streams are not detected by yt-dlp in some streaming websites and there a need to play the streaming video ans the nedd of video extension add-on to anazlyze the web page to detect embeded video streams, we hope that our beloved DownZemAll add these 2 video extensions which are open source and modify them to integrate smoothly with DownZemAll in order to detect enven more streams and send to the download manger !
Thank you ! 

https://github.com/rowrawer/stream-detector

https://github.com/chandler-stimson/live-stream-downloader

**Comment by katastrophe92** [comment]
@trimechee Yeah that would be nice. I still do not get it why this amazing software is not that popular. Does everything you want from a download manager + a lot more. Or im wrong, are other better open source download managers like this one?

**Comment by trimechee** [comment]
@katastrophe92 The best are ant download manager and free dowload manager, that's the resulat of my research, fdm has problem with edge browser, ant is the best and free but the free version doesn't allow to downlader the very high definition video

**Comment by katastrophe92** [comment]
Hi. ADM is not open source
FRD it is open source but scetchy AF. Couldn't find how to download the source code and all the adds on the site.
